Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -115,7 +115,7 @@ public async Task<IReadOnlyList<PackageInfo>> GetAvailablePackages(CancellationT
             using (var response = await _httpClient.SendAsync(
                 new HttpRequestOptions
                 {
-                    Url = "https://repo.jellyfin.org/releases/plugin/manifest.json",
+                    Url = "https://repo.jellyfin.org/releases/plugin/manifest-water.json",
```

Comment:
eh?

Reply:
There are breaking changes to the manifest, so we'll have to switch to a new location. I chose a random name to append to the existing one.

Reply:
Is this a URL for nightly plugins? If not, why not `manifest-stable`?